poweractions/poweractions.py

```diff
@@ -1,7 +1,7 @@
 import asyncio
 import base64
 import aiohttp
-from typing import Any, Optional
+from typing import Any, Optional, Union
```

Comment:
Which means you can remove this diff.
```suggestion
from typing import Any, Optional
```